core/src/main/scripts/importer/validateData.py

```diff
@@ -1009,8 +1012,8 @@ def checkHeader(self, cols):
                 extra={'line_number': self.line_number})
         elif not 'swissprot_identifier' in self.meta_dict:
             self.logger.warning(
-                "A SWISSPROT column was found in a file without an "
-                "associated 'swissprot_identifier' metadatum, assuming "
+                "A SWISSPROT column was found without an associated "
+                "'swissprot_identifier' attribute; assuming "
```

Comment:
"...attribute in the mutation meta file".... i.e. just using another word will not make it clearer. Add some more info.